lib/subscribers/google-genai/config.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const generateContentInternal = {
+  path: './google-genai/generate-content.js',
+  instrumentations: [
+    {
+      channelName: 'nr_generateContentInternal',
+      module: { name: '@google/genai', versionRange: '>=1.1.0 <1.5.0 || >=1.5.1', filePath: 'dist/node/index.cjs' },
```

Comment:
shouldn't this just be `>=1.1.0`? I get 1.5.0 broke our tests but the structures of objects should be the same, right?